languages/csharp/exercises/concept/nullability/.docs/instructions.md

```diff
@@ -0,0 +1,30 @@
+In this exercise you'll be writing code to helping printing name
+badges for factory employees.
+
+Employees have an ID, name and department name. Badge labels are
+formatted as follows: `"[id] - [name] - [DEPARTMENT]"`
+
+### 1. Generate an employee's badge label
+
+Implement the `Badge.Label()` method to return an employee's badge label:
+
+```csharp
+Badge.Label(734, "Ernest Johnny Payne", "Strategic Communication");
+// => "[734] - Ernest Johnny Payne - STRATEGIC COMMUNICATION"
+```
+
+### 2. Deal with the optional parameters on the badge labels
+
+The ID and department name are optional. If someone does not have an
```

Comment:
Could you add an example of what a call would look like and what it would return?

```csharp
Badge.Label(null, "Ernest Johnny Payne", null);
// => "Ernest Johnny Payne - GUEST"
```